tools/travis/setupPrereq.sh

```diff
@@ -32,5 +33,8 @@ $ANSIBLE_CMD couchdb.yml
 $ANSIBLE_CMD initdb.yml
 $ANSIBLE_CMD wipe.yml
 
-$ANSIBLE_CMD properties.yml -e manifest_file="$RUNTIMES_MANIFEST"
+$ANSIBLE_CMD properties.yml \
+  -e "{ \"manifest_file\": \"$RUNTIMES_MANIFEST\",
+      \"invoker_use_runc\": false,
```

Comment:
is `invoker_use_runc` required here?

Reply:
i dont see why runc should be in here either.